homeassistant/components/homematic/climate.py

```diff
@@ -140,8 +141,11 @@ def set_hvac_mode(self, hvac_mode):
         """Set new target hvac mode."""
         if hvac_mode == HVAC_MODE_AUTO:
             self._hmdevice.MODE = self._hmdevice.AUTO_MODE
-        else:
+        elif hvac_mode == HVAC_MODE_HEAT:
             self._hmdevice.MODE = self._hmdevice.MANU_MODE
```

Comment:
Does setting the mode implicitly turn it on if it was previously off?

Reply:
sure